components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <section class="providers-list">
+    <h4>Providers</h4>
+    <ul class="providers-list__list">
+      <template v-for="(item, index) in providersList">
+        <li
+          v-if="item.docsCta && item.docsCta.url"
+          :key="index"
+          class="providers-list__list-item"
+          :class="activeIndex === index ? 'providers-list__list-item_active' : 'providers-list__list-item_inactive'"
+          tabindex="0"
+          @click="updateSelectedProvider(index)"
+          @keydown.enter="updateSelectedProvider(index)"
+        >
+          {{ item.title }}
+          <AppCta
+            class="providers-list__list__cta"
+            v-bind="{ url: item.docsCta.url, label: `Learn more` }"
```

Comment:
Passing these attributes individually seems cleaner:

```suggestion
            label="Learn more"
            :url="item.docsCta.url"
```